config/mcp_config.py

```
@@ -0,0 +1,81 @@
+    def __init__(self):
+        self.config_path = Path.home() / ".claude" / "mcp_config.json"
+        self.default_config = {
+            "endpoints": {
+                "mcp_server": "stdio",  # Use stdio transport for Claude CLI
+                "backup_server": "http://localhost:8080",
+            },
+            "servers": {
+                "self-correcting-executor": {
+                    "command": "python3",
+                    "args": [
+                        str(Path(__file__).parent.parent / "mcp_server" / "main.py")
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)